src/components/appRouter.js

```diff
@@ -124,7 +124,7 @@ class AppRouter {
                 isBack: action === Action.Pop
             });
         } else {
-            console.info('[appRouter] "%s" route not found', normalizedPath, location);
+            // The route is not registered here, so it should be handled by react-router
```

Comment:
All routes should be handled by react-router now, so no need to log ones that are not handled by app router.